### PR TITLE
Add a rule_group_samples metric

### PR DESCRIPTION
**Pull request by prymitive**
This new metric allows tracking how many samples did each rule group generate.

Signed-off-by: Łukasz Mierzwa <l.mierzwa@gmail.com>

### CODE REVIEW

File: rules/manager.go [hunk 1]
```diff
@@ -146,6 +147,14 @@ func NewGroupMetrics(reg prometheus.Registerer) *Metrics {
 			},
 			[]string{"rule_group"},
 		),
+		groupSeries: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: namespace,
+				Name:      "rule_group_series",
+				Help:      "The number of series returned for given rule.",
```
Comment (brian-brazil) [inline rules/manager.go:154]: Rules produce samples, not series.

File: rules/manager.go [hunk 2]
```diff
@@ -146,6 +147,14 @@ func NewGroupMetrics(reg prometheus.Registerer) *Metrics {
 			},
 			[]string{"rule_group"},
 		),
+		groupSamples: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: namespace,
+				Name:      "rule_group_samples",
+				Help:      "The number of samples returned for given rule group.",
```
Comment (brian-brazil) [inline rules/manager.go:154]: This should mention that it was at the last evaluation, like the others do.

Comment (prymitive) [reply rules/manager.go]: Is `rule_group_last_evaluation_samples` ok?

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**